docs/tables/hibp_password.md

```diff
@@ -6,16 +6,24 @@ You must include at _least_ 5 characters in the `prefix` field.
 
 ## Examples
 
-### Show the number of times this password has been compromised
+### List the number of times a password hash has been compromised
 
 ```sql
-select * from hibp_password
- where hash = '5BAA61E4C9B93F3F0682250B6CF8331B7EE68FD8'
+select
+  *
```

Comment:
Can you please replace `*` with either all named columns, or the important ones?